params/config.go

```diff
@@ -845,16 +834,13 @@ func (c *ChainConfig) checkCompatible(newcfg *ChainConfig, headNumber *big.Int,
 	if isForkTimestampIncompatible(c.VerkleTime, newcfg.VerkleTime, headTimestamp) {
 		return newTimestampCompatError("Verkle fork timestamp", c.VerkleTime, newcfg.VerkleTime)
 	}
-	if isForkBlockIncompatible(c.TheogonyBlock, newcfg.TheogonyBlock, headNumber) {
-		return newBlockCompatError("Story Theogony fork block", c.TheogonyBlock, newcfg.TheogonyBlock)
-	}
 	return nil
 }
 
 // BaseFeeChangeDenominator bounds the amount the base fee can change between blocks.
-func (c *ChainConfig) BaseFeeChangeDenominator(num *big.Int) uint64 {
```

Comment:
I think we don't need a config for gas denom since the denom change can only go with hardfork. So the logic here is going to be a different value per hardfork. It cannot be controlled by a config.